### PR TITLE
Updated test for step 19

### LINKED ISSUE #53755: Update test for step 19 of authors project
### Describe the Issue

A camper on the forum was unable to pass with the following code:
https://forum.freecodecamp.org/t/learn-fetch-and-promises-by-building-an-fcc-authors-page-step-19/672642

```js
 <a href="${url}" class="author-link" target="_blank">
          ${author}'s author page
        </a>
```
This is was with this test
````js
```js
assert.equal(document.querySelector('a')?.textContent, `${authorDataArr[0].author}'s author page`);
```
````
Adding the trim method here will fix this

## updated test

````
```js
assert.equal(document.querySelector('a')?.textContent.trim(), `${authorDataArr[0].author}'s author page`);
```
````

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-fetch-and-promises-by-building-an-fcc-authors-page/step-19

### Your code

see explanation above

### Expected behavior

see explanation above

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by shivasaikrishnagadeela**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #53755

### CODE REVIEW

**Review by majestic-owl448 — APPROVED** [review]
Thank you for your contribution!

Two approving reviews are needed before a PR can be merged.

**Comment by Sembauke** [comment]
As a new contributor, we encourage you to read our [contributing guidelines](https://contribute.freecodecamp.org).

We expect our contributors to be aware of the process specific to this project. Following the guidelines religiously earns you the respect of fellow maintainers and saves everyone time.

Some examples of this are:

1. Do not edit files directly through GitHub – while you can, it's not a good idea.
2. Make sure you follow the PR checklist and not just tick things off; otherwise, we won't take you seriously.
3. Use the correct way to link issues in the description of the PR by updating the `XXXXXX`. Do not just add issue numbers everywhere and anywhere you feel like.
4. Keep mentions and review requests to a minimum. We understand you are excited about contributing, and our maintainers will get back to you as soon as they get a chance.
5. Do not work directly off your `main` branch - create a new branch for the changes you are working on.

We appreciate you taking the time to help us, and we hope to see more contributions from you.

Happy Contributing.

**Pull request closed without merging**
